[DO NOT MERGE] Argentine confederation rework

anzu-3278 opened the pull request.

GeneralVonGelre commented on TGC/events/ARGFlavor.txt:
```diff
@@ -21,6 +21,7 @@
 				has_country_flag = Alberdi
 			}
 			war = no
+			is_ideology_enabled = anarcho_liberal
```
This needs to be removed, An-Lib is active from gamestart now

GeneralVonGelre commented on TGC/events/ARGFlavor.txt:
```diff
@@ -105,6 +106,7 @@
 				tag = LPL
 			}
 			year = 1844
+			is_ideology_enabled = anarcho_liberal
```
Same as above

GeneralVonGelre commented on TGC/events/ARGFlavor.txt:
```diff
@@ -1708,214 +1710,214 @@
-			}
-		}
-
+#		#Description: Urquiza's Statement - issue a statement of independence if Rosas doesn't step down
```
Is this intentionally disabled?

ColdSlav replied:
ARG content was intentionally disabled until the ARC rework is finished

GeneralVonGelre commented on TGC/history/pops/1836.1.1/Egypt.txt:
```diff
@@ -205,24 +205,6 @@
         religion = orthodox
         size = 1025
     }
-
```
Are these hungarians going poof intentionally?

GeneralVonGelre commented on TGC/history/pops/1836.1.1/Libya.txt:
```diff
@@ -1,118 +1,138 @@
-#Tripoli-Libya(600000)
-#Tripolitania Region (335000)
-#Tripoli (75000/30000 POPS)
+
```
Indentation needs fixing

GeneralVonGelre commented on TGC/history/pops/1836.1.1/Turkey.txt:
```diff
@@ -19,7 +19,6 @@
         religion = sunni
         size = 175
     }
-
```
Are these empty lines removed intentionally? Otherwise restore

GeneralVonGelre commented on TGCHFMStyleCultureColors/common/cultures.txt:
```diff
@@ -1170,6 +1170,13 @@
 			last_names = { Alfaro Andueza Arboleda Blanco Bonnet Boyd Caama�o Carre�o Carri�n Castro Chiari Cordero Crespo Daza "de Herrera" "de Mosquera" "de Obald�a" "de Olmedo" "de Veintemilla" "de Villamil" Falc�n Flores Garc�a G�mez Gonz�lez Gual Guti�rrez Herr�n Hurtado L�pez Mallarino Manjarr�s Mari�o Melo Monagas Moreno N��ez Obando P�ez Porras Reyes Robles Rocafuerte Rojas Salgar Santander Souble ...
 		}
 
+		# Spanish Gaucho #
```
This file needs to be brought in line with main cultures.txt

GeneralVonGelre commented on TGC/map/province_flag_sprites/kiel_canal.txt:
```diff
@@ -1,3 +1,3 @@
 x = -1
-y = 5
\ No newline at end of file
+y = 4
```
Is this intentional?

GeneralVonGelre commented on Tracked issues and suggestions.txt:
```diff
@@ -15,6 +15,8 @@
 
 	# French literacy low compared to HFM
 
+	# Multiple issues with Switzerland partition related to different Italy tags
```
Many issues that were since fixed have returned here, remove them

GeneralVonGelre commented on TGC/localisation/00_countries-demonyms.csv:
```diff
@@ -893,22 +949,20 @@ VNZ_ADJ;Venezuelan;V
 VOC_ADJ;Compagnie;;;;;;;;;;;;;x
 VOC_colonial_company_ADJ;Dutch;;;;;;;;;;;;;x
 VOJ_ADJ;Vojvodinian;;;;;;;;;;;;;x
-VOJ_colonial_company_ADJ;Serbian Frontiersman;;;;;;;;;;;;;x
```
Why are these going poof?

GeneralVonGelre commented on TGC/localisation/00_countries-demonyms.csv:
```diff
@@ -893,22 +949,20 @@ VNZ_ADJ;Venezuelan;V
 WES_ADJ;Westphalian;;;;;;;;;;;;;x
 WHA_ADJ;Welsh;;;;;;;;;;;;;x
 WIA_ADJ;Vientiane;;;;;;;;;;;;;x
-WIN_ADJ;Windward Islander;;;;;;;;;;;;;x
```
Is this one removed intentionally?

ColdSlav committed:
Delete 4008 - La  Nacional.txt

ColdSlav committed:
Delete 4007 - Laguna Blanca.txt

ColdSlav committed:
Delete 4006 - Llica.txt

ColdSlav committed:
Delete 4005 - Huacareta.txt

ColdSlav committed:
Delete 4004 - Igueroj.txt

ColdSlav committed:
Delete 4003 - La Carlota.txt

ColdSlav committed:
Delete 4002 - Serrezuela.txt

ColdSlav committed:
Delete 4001 - Laguna Brava.txt

ColdSlav committed:
Delete 4000 - Los Erizos.txt

ColdSlav committed:
Delete 3998 - Villalonga.txt

ColdSlav committed:
Delete 3999 - Villa Dolores.txt

ColdSlav committed:
South America Province Files

ColdSlav committed:
Rename 3988 - Serrezuela.txt to 3987 - Serrezuela.txt

ColdSlav committed:
Merge pull request #227 from ColdSlav/Argentine-Confederation-Rework

Update Province History Files

ColdSlav committed:
Update Argentina.txt

ColdSlav committed:
Update Bolivia.txt

ColdSlav committed:
Update Country History Files

ColdSlav committed:
Delete Arg Beni.txt

ColdSlav committed:
Delete Arg Chuquisaca.txt

ColdSlav committed:
Delete Arg Cochabamba.txt

ColdSlav committed:
Delete Arg La Paz.txt

ColdSlav committed:
Delete Arg Oruro.txt

ColdSlav committed:
Delete Arg Santa Cruz.txt

ColdSlav committed:
Delete Arg Tarija.txt

ColdSlav committed:
Delete Arg Potosi.txt

ColdSlav committed:
Update countries.txt

ColdSlav committed:
Delete ABN - Arg Beni.txt

ColdSlav committed:
Delete ACB - Arg Cochabamba.txt

ColdSlav committed:
Delete AOU - Arg Oruro.txt

ColdSlav committed:
Delete APS - Arg Potosi.txt

ColdSlav committed:
Delete APZ - Arg La Paz.txt

ColdSlav committed:
Delete AQS - Arg Chuquisaca.txt

ColdSlav committed:
Delete ASC - Arg Santa Cruz.txt

ColdSlav committed:
Delete ATR - Arg Tarija.txt

ColdSlav committed:
Delete ABN.tga

ColdSlav committed:
Delete ABN_fascist.tga

ColdSlav committed:
Delete ABN_communist.tga

ColdSlav committed:
Delete ABN_monarchy.tga

ColdSlav committed:
Delete ABN_monarchy2.tga

ColdSlav committed:
Delete ABN_republic.tga

ColdSlav committed:
Delete ABN_republic2.tga

ColdSlav committed:
Delete ACB_communist.tga

ColdSlav committed:
Delete ACB.tga

ColdSlav committed:
Delete ACB_fascist.tga

ColdSlav committed:
Delete ACB_monarchy2.tga

ColdSlav committed:
Delete ACB_monarchy.tga

ColdSlav committed:
Delete ACB_republic.tga

ColdSlav committed:
Delete ACB_republic2.tga

ColdSlav committed:
Delete AOU.tga

ColdSlav committed:
Delete AOU_communist.tga

ColdSlav committed:
Delete AOU_fascist.tga

ColdSlav committed:
Delete AOU_monarchy.tga

ColdSlav committed:
Delete AOU_monarchy2.tga

ColdSlav committed:
Delete AOU_republic.tga

ColdSlav committed:
Delete AOU_republic2.tga

ColdSlav committed:
Delete APS_communist.tga

ColdSlav committed:
Delete APS.tga

ColdSlav committed:
Delete APS_fascist.tga

ColdSlav committed:
Delete APS_monarchy2.tga

ColdSlav committed:
Delete APS_monarchy.tga

ColdSlav committed:
Delete APS_republic2.tga

ColdSlav committed:
Delete APS_republic.tga

ColdSlav committed:
Delete APZ.tga

ColdSlav committed:
Delete APZ_communist.tga

ColdSlav committed:
Delete APZ_fascist.tga

ColdSlav committed:
Delete APZ_monarchy.tga

ColdSlav committed:
Delete APZ_monarchy2.tga

ColdSlav committed:
Delete APZ_republic.tga

ColdSlav committed:
Delete APZ_republic2.tga

ColdSlav committed:
Delete AQS.tga

ColdSlav committed:
Delete AQS_fascist.tga

ColdSlav committed:
Delete AQS_communist.tga

ColdSlav committed:
Delete AQS_monarchy.tga

ColdSlav committed:
Delete AQS_republic.tga

ColdSlav committed:
Delete AQS_monarchy2.tga

ColdSlav committed:
Delete AQS_republic2.tga

ColdSlav committed:
Delete ASC_communist.tga

ColdSlav committed:
Delete ASC.tga

ColdSlav committed:
Delete ASC_fascist.tga

ColdSlav committed:
Delete ASC_monarchy2.tga

ColdSlav committed:
Delete ASC_monarchy.tga

ColdSlav committed:
Delete ASC_republic.tga

ColdSlav committed:
Delete ASC_republic2.tga

ColdSlav committed:
Delete ATR.tga

ColdSlav committed:
Delete ATR_communist.tga

ColdSlav committed:
Delete ATR_fascist.tga

ColdSlav committed:
Delete ATR_monarchy.tga

ColdSlav committed:
Delete ATR_monarchy2.tga

ColdSlav committed:
Delete ATR_republic.tga

ColdSlav committed:
Delete ATR_republic2.tga

ColdSlav committed:
Delete Diaguita.txt

ColdSlav committed:
Add files via upload

ColdSlav committed:
Delete Cuyo.txt

ColdSlav committed:
Delete DGT_fascist.tga

ColdSlav committed:
Delete DGT_republic.tga

ColdSlav committed:
Delete DGT_monarchy.tga

ColdSlav committed:
Delete DGT_communist.tga

ColdSlav committed:
Delete DGT_monarchy2.tga

ColdSlav committed:
Delete DGT_republic2.tga

ColdSlav committed:
Delete DGT.tga

ColdSlav committed:
Delete DGT - Diaguita.txt

ColdSlav committed:
Delete CUY.tga

ColdSlav committed:
Delete CUY_monarchy.tga

ColdSlav committed:
Delete CUY_fascist.tga

ColdSlav committed:
Delete CUY_republic.tga

ColdSlav committed:
Delete CUY_monarchy2.tga

ColdSlav committed:
Delete CUY_communist.tga

ColdSlav committed:
Delete CUY_republic2.tga

ColdSlav committed:
Delete CUY - Cuyo.txt

ColdSlav committed:
Update countries.txt

ColdSlav committed:
Delete Misiones.txt

ColdSlav committed:
Delete MSN - Misiones.txt

ColdSlav committed:
Delete MSN.tga

ColdSlav committed:
Delete MSN_fascist.tga

ColdSlav committed:
Delete MSN_monarchy.tga

ColdSlav committed:
Delete MSN_republic.tga

ColdSlav committed:
Delete MSN_communist.tga

ColdSlav committed:
Delete MSN_monarchy2.tga

ColdSlav committed:
Delete MSN_republic2.tga

ColdSlav committed:
Delete REB _fascist.tga

ColdSlav committed:
Delete REB _republic.tga

ColdSlav committed:
Delete REB _republic2.tga

ColdSlav committed:
THOSE GOSH DARN REBELS

ColdSlav committed:
Update cultures.txt

ColdSlav committed:
Update PuppetStates.txt

ColdSlav committed:
Update 604 - Basel.txt

ColdSlav committed:
Delete 3274 - Svaneti.txt

ColdSlav committed:
Delete 3382 - Gazakh.txt

ColdSlav committed:
Delete 3430 - Kabuchi.txt

ColdSlav committed:
Delete 3667 - Kislovodsk.txt

ColdSlav committed:
Delete 3670 - Kishen-Aukh.txt

ColdSlav committed:
Delete 3679 - Khasavyurt.txt

ColdSlav committed:
Delete 3681 - Terekli-Mekteb.txt

ColdSlav committed:
Delete 3682 - Khamish-Burun.txt

ColdSlav committed:
Delete 3687 - Dzheyrakh.txt

ColdSlav committed:
Delete 3690 - Bekovichi.txt

ColdSlav committed:
Delete 3694 - Soleny.txt

ColdSlav committed:
Delete 3696 - Medvezhye.txt

ColdSlav committed:
Delete 3708 - Ag-Taklya.txt

ColdSlav committed:
Delete 3711 - Vartashen.txt

ColdSlav committed:
Delete 3714 - Qusar.txt

ColdSlav committed:
Delete 3720 - Khosta.txt

ColdSlav committed:
Delete 3722 - Gorio.txt

ColdSlav committed:
Delete 3733 - Javad.txt

ColdSlav committed:
Delete 3745 - Metskhal.txt

ColdSlav committed:
Delete 3739 - Karyagino.txt

ColdSlav committed:
Delete 3747 - Tarskaya.txt

ColdSlav committed:
Delete 3751 - Khachmaz.txt

ColdSlav committed:
Delete 3752 - Izmayili.txt

ColdSlav committed:
Delete 3755 - Beyuk-Vedi.txt

ColdSlav committed:
Delete 3758 - Tsageri.txt

ColdSlav committed:
Delete 3759 - Ispir.txt

ColdSlav committed:
Delete 3762 - Sarybaba.txt

ColdSlav committed:
Restore Argentinian Country Files

ColdSlav committed:
Restore Flag FIles

ColdSlav committed:
Restore ATR - Arg Tarija.txt

ColdSlav committed:
Update countries.txt

ColdSlav committed:
Restore Country Files Part Two

ColdSlav committed:
New Country Colors for the Caucasus

ColdSlav committed:
Update cultures.txt

ColdSlav committed:
Restore CUY - Cuyo.txt

ColdSlav committed:
Restore CUY Flags

ColdSlav committed:
Restore Cuyo

ColdSlav committed:
Argentina is no Longer Releasable

joelAVS committed:
Updated map

ColdSlav committed:
Tarija and Bolivia Now Have Proper Cores

ColdSlav committed:
New Province in Santa Fe

ColdSlav committed:
Update pop_types.txt

ColdSlav committed:
Update pop_types.txt

ColdSlav committed:
Update pop_types.txt

ColdSlav committed:
Influence Modifiers

ColdSlav committed:
New Flags for Buenos Aires (SBA)

GeneralVonGelre committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Event Modifiers Adjustments

ColdSlav committed:
First Batch of Decisions

No localization for now, for testing purposes only.

ColdSlav committed:
First Batch of Events

No localization for now, for testing purposes only.

ColdSlav committed:
Decision Pictures

ColdSlav committed:
Event Pictures

ColdSlav committed:
Decision Adjustments

ColdSlav committed:
Event Adjustments

ColdSlav committed:
Merge pull request #238 from ColdSlav/Argentine-Confederation-Rework

First Batch of Content for Argentina [Check description]

ColdSlav committed:
New Province Picture for Buenos Aires

joelAVS committed:
Update ARGCivilWar.txt

joelAVS committed:
typo ops

ColdSlav committed:
Smol Adjustments to SBA - Buenos Aires

GeneralVonGelre committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Merge pull request #250 from rderekp/master

Update Argentine Confederation Rework

GeneralVonGelre committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Added Missing Theocracy Flags Part One

ColdSlav committed:
Added Missing Theocracy Flags Part Two

ColdSlav committed:
Modifiers Adjustments

ColdSlav committed:
New CBs for Unitarians and Federalists

ColdSlav committed:
Setup Decision Adjustments

ColdSlav committed:
Argentine Confederation Decision Adjustments

ColdSlav committed:
New Events

;)

ColdSlav committed:
Added Missing Province History Files

ColdSlav committed:
New Newspaper Title

ColdSlav committed:
Part Two

ColdSlav committed:
Localization Adjustments

ColdSlav committed:
Added Missing Flag

ColdSlav committed:
I Forgor

ColdSlav committed:
New Leaders Part One

ColdSlav committed:
New Leaders Part Two

ColdSlav committed:
Even more leaders

ColdSlav committed:
New Country Leaders, Unit and Relation Adjustments

ColdSlav committed:
Adjustments

ColdSlav committed:
More Adjustments

ColdSlav committed:
Country History Adjustments

ColdSlav committed:
Casus Belli Adjustments

ColdSlav committed:
Merge pull request #262 from rderekp/master

Update Argentina

ColdSlav committed:
New Casus Belli ;)

ColdSlav committed:
Argentine Dismantlement of Bolivia - Initial Draft

joelAVS committed:
New mod path

ColdSlav committed:
Smol Adjustment

ColdSlav committed:
New Regions

ColdSlav committed:
Some Adjustments

ColdSlav committed:
More Adjustments

ColdSlav committed:
Setup Adjustments

ColdSlav committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Adjustments and Fixes

ColdSlav committed:
Some Map Adjustments

ColdSlav committed:
Map Adjustments

ColdSlav committed:
Uruguay PoPs

ColdSlav committed:
Caucasus PoPs 1

ColdSlav committed:
Caucasus PoPs 2

ColdSlav committed:
Caucasus PoPs 3

ColdSlav committed:
Caucasus PoPs 4

ColdSlav committed:
Rio Grande do Sul Now Has Mil. Access to Uruguay

ColdSlav committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
New Country Colors

ColdSlav committed:
Culture Colors Adjustments

ColdSlav committed:
PoP Data Finished

ColdSlav committed:
Fixes n' Adjustments

- Adjusted some regions
- Temporarily disabled the Caucasus Viceroyalty Decision
- Added a "Platinean Breadbasket" modifier
- Fixed event with no MTTH that was causing crashes

GeneralVonGelre committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Update Old Events to Fit With New Provinces Pt. 1

ColdSlav committed:
Updates, Adjustments, and All That

ColdSlav committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Fixes

ColdSlav committed:
Life Rating Adjustments

ColdSlav committed:
& More Fixes

ColdSlav committed:
Fixes Fixes Fixes

ColdSlav committed:
Removed Racist Term

Fuck you Fraggy you should've never joined the team

ColdSlav committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Revert "Merge branch 'master' into Argentine-Confederation-Rework"

This reverts commit 4bc81a8931afe535622c8fa17818e2dcf72f6067, reversing
changes made to 57b5a06e15a9225c7a9c1344bd9972356be1f4a1.

ColdSlav committed:
Map and Other Adjustments

ColdSlav committed:
Merge branch 'master' into Argentine-Confederation-Rework

ColdSlav committed:
Smol detail

ColdSlav committed:
Continent Fixes & Adjustments

ColdSlav committed:
Caucasus Adjustments
